db/pf-schema-X.Y.Z.sql

```diff
@@ -1114,6 +1114,7 @@ CREATE TABLE radius_audit_log (
   KEY `created_at` (created_at),
   KEY `mac` (mac),
   KEY `ip` (ip),
+  KEY `auth_status` (auth_status, created_at),
```

Comment:
Add this after user_name so the upgraded schema and the a new created schemas are the same.